packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -108,12 +106,12 @@ export default function CreatePatternModal( {
 						help={ __(
 							'Editing the pattern will update it anywhere it is used.'
 						) }
-						checked={ ! syncType }
+						checked={ syncType === PATTERN_SYNC_TYPES.full }
```

Comment:
This is not the same as `!syncType`. IIRC, fully synced patterns have empty `syncStatus` for backward-compatible reason? I think this only matters when we create the pattern and if it handles `'fully'` or `''` in `onCreate` 🤔 . @glendaviesnz or @aaronrobertshaw might have some insights on this.

Reply:
Good catch, I missed it in my review 😥 

We should maintain the same logic.

Reply:
correct, but the only place where it is actually set to undefined is in the [action here](https://github.com/WordPress/gutenberg/pull/54484/files#diff-73a5761acf8f02c4c35123c516f20245ac6cf923e4f350f96b90ff7f184327f6R26), and that still works as expected as everything except syncType of `unsynced` is set there as undefined.

Reply:
Thanks for catching that. That was a bad rebase on my part probably. I'll fix it up.

Reply:
Oh wait, was in f54bf749cc80d86efe8eac29cc1286b52a532d3a 😄

Reply:
> correct, but the only place where it is actually set to undefined is in the [action here](https://github.com/WordPress/gutenberg/pull/54484/files#diff-73a5761acf8f02c4c35123c516f20245ac6cf923e4f350f96b90ff7f184327f6R26), and that still works as expected as everything except syncType of unsynced is set there as undefined.


Oh, sorry I just saw this. No need to revert?

Reply:
Yeah @glendaviesnz is correct. I don't think this introduces any unexpected behaviors. As a matter of fact, I think this is better than undefined as it's explicit and it matches the types in the actions. 👍